Library/Homebrew/cmd/audit.rb

```diff
@@ -628,7 +628,7 @@ def audit_revision
 
     fv = FormulaVersions.new(formula, :max_depth => 10)
     revision_map = fv.revision_map("origin/master")
-    if (revisions = revision_map[formula.version]).any?
+    if !(revisions = revision_map[formula.version]).empty?
```

Comment:
Can you move the assignment before the `if`? Assignments in conditions can be easily misunderstood to be a comparison or missed, particularly when they are hidden in surrounding logic like in this case.